metric/importer/prometheus.py

```diff
@@ -57,53 +56,39 @@ def file_list(dir=None):
             elif file.endswith('.dat'):
                 raw = zlib.decompress(fileopt.read_file(file, 'rb'))
             else:
-                logging.debug("Skipped unrecorgnized file '%s'" % file)
+                logging.debug("Skip unrecorgnized file '%s'" % file)
                 continue
             yield json.loads(raw)
 
-    def build_series(self):
-        def format_prom_metric(key=None):
-            points = []
-            point = {'fields': {}}
-            # build point header
-            for metric in key:
-                point['measurement'] = metric['metric']['__name__']
-                point['tags'] = {
-                    'cluster': self.db_name,
-                    'monitor': 'prometheus',
-                }
-                for k, v in metric['metric'].items():
-                    point['tags'][k] = v
-                # build point values
-                for value in metric['values']:
-                    point['time'] = datetime.datetime.utcfromtimestamp(
-                        value[0]).strftime('%Y-%m-%dT%H:%M:%SZ')
-                    try:
-                        point['fields']['value'] = float(value[1])
-                    except ValueError:
-                        point['fields']['value'] = value[1]
-                    points.append(point.copy())
-            return points
+    def exec_importer(self, data, chunk_size=2000):
+        base_dir = os.path.join(util.pwd(), "../")
+        importer = os.path.join(base_dir, "bin/prom2influx")
+        cmd = [importer,
+               "-db", self.db_name,
+               "-host", self.host,
+               "-port", "%s" % self.port,
+               "-chunk", "%s" % chunk_size,  # chunk size of one write request
+               ]
+        logging.debug("Running cmd: %s" % ' '.join(cmd))
+        return util.run_cmd(cmd, input=json.dumps(data).encode('utf-8'))
```

Comment:
why not let prom2influx parse json directly, avoid  json.load ->  json.dumps -> json.Unmarshal